src/views/Auth.vue

```diff
@@ -0,0 +1,54 @@
+<template lang="pug">
+  .nav-bar
+    .items-container
+      img.logo(src='../assets/wolox-logo.png')
+      button.logout-button(@click='logout')
+        | Logout
+
+</template>
+
+<script>
+import { deleteCurrentSession } from '@/services/auth'
+
+export default {
+  name: 'auth',
+  methods: {
+    logout () {
+      deleteCurrentSession()
+      this.$router.push('/login')
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../scss/commons/colors.scss";
+
+.nav-bar {
+  background-color: $white;
+  border-top: 5px solid $scooter;
+  box-shadow: 0 3px 5px $shadow;
+  width: 100vw;
+
+  .items-container {
+    display: flex;
+    height: 100%;
+    justify-content: space-between;
+    margin: 0 auto;
```

Comment:
why this auto?

Reply:
As I set a max-width, I want this component to be centered without adding a `display:flex; justify-content: center;` to its parent.

Reply:
It's recommendable to set alignments in the parent node